pkg/store/experiment/experiment.go

```diff
@@ -42,6 +42,8 @@ func (e *experimentStore) Find(context.Context, int64) (*core.ArchiveExperiment,
 func (e *experimentStore) FindByName(context.Context, string, string) (*core.ArchiveExperiment, error) {
 	return nil, nil
 }
-func (e *experimentStore) Create(context.Context, *core.ArchiveExperiment) error { return nil }
-func (e *experimentStore) Update(context.Context, *core.ArchiveExperiment) error { return nil }
```

Comment:
I delete this since I think it should not be changeable for history experiments.